pkg/storage/stores/tsdb/head.go

```diff
@@ -58,55 +62,30 @@ guaranteeing we maintain querying consistency for the entire data lifecycle.
 
 // TODO(owen-d)
 type Metrics struct {
-	seriesNotFound                prometheus.Counter
-	tsdbManagerUpdatesTotal       prometheus.Counter
-	tsdbManagerUpdatesFailedTotal prometheus.Counter
-	tsdbHeadRotationsTotal        prometheus.Counter
-	tsdbHeadRotationsFailedTotal  prometheus.Counter
-	tsdbWALTruncationsTotal       prometheus.Counter
-	tsdbWALTruncationsFailedTotal prometheus.Counter
-	tsdbCreationsTotal            *prometheus.CounterVec
-	tsdbCreationsFailedTotal      *prometheus.CounterVec
+	seriesNotFound prometheus.Counter
+	headRotations  *prometheus.CounterVec
+	walTruncations *prometheus.CounterVec
+	tsdbCreations  *prometheus.CounterVec
 }
 
 func NewMetrics(r prometheus.Registerer) *Metrics {
 	return &Metrics{
 		seriesNotFound: promauto.With(r).NewCounter(prometheus.CounterOpts{
 			Name: "loki_tsdb_head_series_not_found_total",
-			Help: "Total number of requests for series that were not found.",
-		}),
-		tsdbManagerUpdatesTotal: promauto.With(r).NewCounter(prometheus.CounterOpts{
-			Name: "loki_tsdb_manager_updates_total",
-			Help: "Total number of tsdb manager updates (loading/rotating tsdbs in mem)",
+			Help: "Total number of requests for series that were not found",
 		}),
-		tsdbManagerUpdatesFailedTotal: promauto.With(r).NewCounter(prometheus.CounterOpts{
-			Name: "loki_tsdb_manager_updates_failed_total",
-			Help: "Total number of tsdb manager update failures (loading/rotating tsdbs in mem)",
-		}),
-		tsdbHeadRotationsTotal: promauto.With(r).NewCounter(prometheus.CounterOpts{
+		headRotations: promauto.With(r).NewCounterVec(prometheus.CounterOpts{
 			Name: "loki_tsdb_head_rotations_total",
```

Comment:
What do you think about adding `attempts` word to it?
```suggestion
			Name: "loki_tsdb_head_rotation_attempts_total",
```